source/grid.F90

```diff
@@ -1025,6 +1033,133 @@ subroutine init_grid2(errorCode)
 
  end subroutine init_grid2
 
+!***********************************************************************
+!BOP
+! !IROUTINE: horiz_grid_amuse
+! !INTERFACE:
+
+ subroutine horiz_grid_amuse(latlon_only)
```

Comment:
right now it generates a hardcoded test grid, must move to interface (with the same trick as for KMT, see comments on closed #3 )

Reply:
will do this another time